### DISCUSSION TITLE
Option to link Taxonomies in navigation builder?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#5376] [Q&A] [answered]
Hi, 

It'd be super helpful if we could link to Taxonomies and Terms in the navigation builder. 

Something like this would be ideal.
![Screenshot 2022-03-01 at 07 40 03](https://user-images.githubusercontent.com/72254741/156126065-3e0813a7-4610-4d8d-810e-c265b6dece60.png)

The navigation builder allows me to link to any page, which is ok, so it's possible to link to Taxonomy and Term pages. BUT the title and url are set in the nav builder, unlike page entries they don't auto update if a page title or url change.

Could something like this make it into a future release?

**Comment by edalzell** [comment, marked as answer]
Looks like you've already suggested this: https://github.com/statamic/ideas/issues/631 ?
